examples/worlds/lightmap.sdf

```diff
@@ -0,0 +1,107 @@
+<?xml version="1.0" ?>
+<!--
+
+A world demonstrating an indoor environment model that uses a lightmap.
+There are no dynamic lights or shadows in the scene.
+
+-->
+
+<sdf version="1.7">
+  <world name="lightmap">
+    <physics name="1ms" type="ignored">
+      <max_step_size>0.001</max_step_size>
+      <real_time_factor>1.0</real_time_factor>
+    </physics>
+    <plugin
+      filename="libignition-gazebo-physics-system.so"
```

Comment:
Is it possible to remove some of these plugins now that some plugin loading is automatic?